packages/sdk/src/models/index.ts

```diff
@@ -653,7 +653,7 @@ export default class Models {
     const orderByQuery =
       orderBy === "newest" ? `marketId_${orderingStr}` : `end_${orderingStr}`;
 
-    const timestamp = (await this.api.query.timestamp.now()).toNumber();
+    const timestamp = (await this.api.query.timestamp.now()) as any as number;
```

Comment:
I think it's best to use `toString` to actually convert it to something we can use rather than type casting. By using `as any` we lose the benefits of typescript so if the type changes in the future we will only know about it when we have a bug. Using `as number` could also cause issues because we know it is a `Codec` so won't share the same methods as `number`

Reply:
Gotcha, done!